spec/rspec/matchers/dsl_spec.rb

```diff
@@ -31,9 +31,9 @@ def ok
 
   it "clears user instance variables between invocations" do
     RSpec::Matchers.define(:be_just_like) do |expected|
-      match do |actual_value|
+      match do |actual|
         @foo ||= expected
-        @foo == actual_value
+        @foo == actual
```

Comment:
do we need to revert this?

Reply:
The [impact on `rspec-mocks`](https://github.com/rspec/rspec-mocks/commit/3b909ed1a951bbca340ea98c27ab65da7f43881c) was definitely more significant than [here](https://github.com/rspec/rspec-expectations/pull/703/commits/fdd9ef3862c4fa09acef0b369ecad8eb5de72dee).
`actual` is the original naming, and Myron [mentioned](https://bugs.ruby-lang.org/issues/10661) that "changed many variable and method names to names I do not like as much". Since the Ruby warning is no more, I believe it's ok to bring the code to its original state.

I've changed the names along with the removal of redundant `self`s that were introduced in the same commit. WDYT if I revert the changes to variable names in this pull request and address this in a follow-up?

Reply:
I trust your judgement here